cli/src/commands/squash.rs

```diff
@@ -65,15 +64,20 @@ pub(crate) struct SquashArgs {
     /// description(s) of the source revision(s)
     #[arg(long, short, conflicts_with = "message_paragraphs")]
     use_destination_message: bool,
-    /// Interactively choose which parts to squash
+    /// Interactively choose which parts to squash.
+    /// Implies --keep-empty.
     #[arg(long, short)]
     interactive: bool,
     /// Specify diff editor to be used (implies --interactive)
     #[arg(long, value_name = "NAME")]
     tool: Option<String>,
-    /// Move only changes to these paths (instead of all paths)
+    /// Move only changes to these paths (instead of all paths).
+    /// Implies --keep-empty.
```

Comment:
rewording nit to keep it consistent with other args:

```suggestion
    /// Move only changes to these paths instead of all paths (implies --keep-empty)
```

Not too sure what the best way to handle the "double parentheses" is - feel free to rephrase, or ignore this comment.